src/xtgeo/common/xtgeo_dialog.py

```diff
@@ -451,7 +464,7 @@ def testsetup(self):
         return True
 
     @staticmethod
-    def timer(*args):
+    def timer(*args: float) -> float:
```

Comment:
Is float okay here? It seems wrong to widen the typing to `Any` for unpacking, but I have no suggestions

Reply:
I can be, but looking at how the this function is used we will get an empty tuple, or an tuple with one float in it (the timestamp from the previews call to `timer(...)`)